core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/cumSum.kt

```diff
@@ -18,6 +21,36 @@ import kotlin.reflect.typeOf
 
 // region DataColumn
 
+/**
+ * ## The CumSum Operation
+ *
+ * Computes the cumulative sum of the values in the {@get DATA_TYPE}.
```

Comment:
The way it's written now suggests we get a single value result, while we actually we get a new column containing all cumulative sum values (or a df with new columns containing...)